tcp_logger.go

```diff
@@ -83,6 +84,7 @@ func (t *TCPLogger) handleConnError(err error, b []byte) {
 func (t *TCPLogger) RetryConnection() error {
 	conn, err := net.Dial(t.conn.RemoteAddr().Network(), t.conn.RemoteAddr().String())
```

Comment:
Should we log every time a new connection is opened?

Reply:
Sounds good to me.  Added a statement to declare connection reestablish being attempted.